src/object/cli_obj.c

```diff
@@ -1718,27 +1718,23 @@ uint32_t
 dc_obj_retry_delay(tse_task_t *task, int err, uint16_t *retry_cnt, uint16_t *inprogress_cnt,
 		   uint32_t timeout_sec)
 {
-	uint32_t	delay = 0;
-	uint32_t	limit = 4;
+	uint32_t delay = 0;
+
+	if (err == -DER_INPROGRESS || err == -DER_UPDATE_AGAIN)
+		++(*inprogress_cnt);
```

Comment:
inprogress_cnt seems useless?

Reply:
Yeah. I discussed this with @Nasf-Fan and he is fine with it